src/main.py

```diff
@@ -12,21 +14,45 @@
     f"{PATH_TO_DIR}{os.sep}assets{os.sep}menu{os.sep}main_menu_background.png"
 )
 
+"""
```

Comment:
Doc strings should be the first line of the method (to link the doc string to the method.)